### ISSUE TITLE
Screen reader doesn't read label of an attachment control when it gets the focus

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#5188]
[+1 from customer](https://3.basecamp.com/3600924/buckets/2192969/messages/4553596584)
